ephios/core/signup/participants.py

```diff
@@ -49,10 +50,10 @@ def all_participations(self):
 
     @functools.lru_cache(maxsize=64)
```

Comment:
maybe we should get rid of the functools cache because of the potential collision problems? and you are caching the queryset anyways